SampleVIPSwift/SampleVIPSwift/SampleVIPSwift/Scenes/FirstScene/FirstSceneInteractor.swift

```diff
@@ -23,7 +23,7 @@ protocol FirstSceneDataStore {
     var dataSource: FirstSceneModel.DataSource { get }
 }
 
-final class FirstSceneInteractor<Factory>: Interactable, FirstSceneDataStore where Factory: ServicesFactorable & FirstSceneFactorable {
```

Comment:
In the current sample implementation, we are not creating any "Services" so ServiceFactorable is only used as an example of how we can add Services thought Dependency Injection.
Also a "ServiceFactorable" constraint is good at this level to force the "FirstSceneInteractor" to use only functions declared inside "ServiceFactorable" and "FirstSceneFactorable"

PS: I will add some service implementation to the sample, for a better understanding, thanks